lib/gs-app.c

```diff
@@ -4136,6 +4165,7 @@ gs_app_init (GsApp *app)
 	                                    g_free,
 	                                    g_free);
 	priv->allow_cancel = TRUE;
+	priv->cancellable = g_cancellable_new ();
```

Comment:
You could avoid this if you added a `!priv->cancellable ||` condition to the `is_cancelled` check in `gs_app_get_cancellable()`